src/Avalonia.Input/TappedEventArgs.cs

```diff
@@ -6,7 +6,12 @@ namespace Avalonia.Input
     public class TappedEventArgs : RoutedEventArgs
     {
         private readonly PointerEventArgs lastPointerEventArgs;
-
+        public IPointer Pointer => lastPointerEventArgs.Pointer;
+        public InputModifiers InputModifiers => lastPointerEventArgs.InputModifiers;
+        public KeyModifiers KeyModifiers => lastPointerEventArgs.KeyModifiers;
+        public ulong Timestamp => lastPointerEventArgs.Timestamp;
+        public IPointerDevice Device => lastPointerEventArgs.Device;
```

Comment:
InputModifiers and Device are deprecated.
https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Input/PointerEventArgs.cs#L58
https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Input/PointerEventArgs.cs#L61

Reply:
You either should add Obsolete attribute here as well or better just to remove them

Reply:
I'm sorry, I didn't notice that the Device property is obsolete. I removed these properties to avoid increasing the number the deprecated properties